### PR TITLE
IoT-chatbot Documentation

### PR DESCRIPTION
**Pull request by Nageshbansal**
Please review this PR .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,133 @@
-# IoT-ChatBot
-Open Projects 2021
+# ***IoTChatbot***
```
Comment (theobscuredev) [inline README.md]: Add '## Open Projects 2021' after the project title.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,133 @@
...
+![image](https://user-images.githubusercontent.com/76246968/127762969-0b0b08ea-84e5-41cd-a843-ae954c220526.png)
```
Comment (theobscuredev) [inline README.md]: Center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `LED Matrix`
  `<i>LED Matrix</i>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

Also resize a few images by assigning the dimensions.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,133 @@
...
+
+
+## Abstract 
+ It is a chatbot integrated with an Iot (Internet of Things) network of devices. The chatbot acts as an interface for the user to view real time data and live status of home appliances as well as control them by giving commands.
```
Comment (theobscuredev) [inline README.md]: Justify the entire text using the `<p>` paragraph tag and `justify` attribute. Make sure you justify all the paragraphs separately for the entire content in README. For eg:
`<p align="justify">`
  `<h2>Abstract</h2>`
  `<p>A Rube Goldberg Machine is .........................</p>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

File: README.md [hunk 4]
```diff
@@ -1,2 +1,133 @@
...
+
+## Motivation
+Using IoT to allow communication between various home appliances, security systems, lighting, entertainment systems and sensors will enhance lives of people and make homes smarter. Controlling home device via chatbot remotely is convenient, easy to use and time saving.
+
```
Comment (theobscuredev) [inline README.md]: Insert a heading 'Software Aspect' and include Networking and Chatbot under it.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,133 @@
...
+## Chatbot
+  - We used Flask-python Framework for creating Chatbot and Dashboard
+  - We also added auth login system for security measures
+  - Sqlite3 act as an database in backend
+  - ***Microsoft Azure App services act as Cloud based hosting platform*** 
+ 
+## WORKFLOW
+ ### IoT-chatbot Architecture
+
+  ![Screenshot (376)](https://user-images.githubusercontent.com/76246968/127763126-a25261c6-256c-4462-b347-6034bd148757.png)
```
Comment (theobscuredev) [inline README.md]: Include this workflow before 'Software Aspect' and 'Networking' part.

**Review by theobscuredev — CHANGES_REQUESTED** [review]
Hey @Nageshbansal ,

Really nice work team. Just make these minor changes and we're good to go then:

1. Update the README according to the changes suggested. Precisely, focus on center aligning the images and justifying the text. And insert the 'Software Aspect' heading.
2. Change the format of the poster to .jpeg from .jpg
3. Update the name of the folder 'Poster and Report'
4. Try to maintain a uniform naming convention for all the files.

**Comment by theobscuredev** [comment]
@Nageshbansal @vaishnavi-gupta18 Lemme know once you've made the final changes. Make sure to tag me (@) in this thread.

**Comment by Nageshbansal** [comment]
@theobscuredev  we have made the required changes ,  Please review this  PR once again.

**Comment by vaishnavi-gupta18** [comment]
@theobscuredev We're done with the final changes. Please review the PR again.

**Comment by theobscuredev** [comment]
Nice work!
@SanjeevKrishnan I suggest for merge

**Comment by SanjeevKrishnan** [comment]
@Nageshbansal @vaishnavi-gupta18 @khushi861 @Kushagra-Agarwal44 Great Work! 
PR looks perfect. Merging the PR

**Pull request merged**